src/config.rs

```diff
@@ -30,7 +33,18 @@ impl Default for UiConfig {
 }
 
 pub fn load() -> anyhow::Result<Config> {
-    let file = std::fs::read_to_string("settings.toml")?;
+    let workspace = PathBuf::from("settings.toml");
+    let xdg_home = ProjectDirs::from("me", "maxjoehnk", "sidenotes")
+        .unwrap()
```

Comment:
Hier wäre statt `unwrap` `expect` ganz gut damit du eine Fehlermeldung ausgeben kannst.
Also bspw `expect("Home directory could not be detected")`.

Ansonsten könnte man das auch über das Result abbilden damit der Fehler zukünftig in der Oberfläche abgebildet werden kann. Das kann ich aber auch später anpassen.

Reply:
Done